community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -214,16 +275,16 @@ class QueryPlanTest extends DocumentingTestBase {
     )
   }
 
-  @Test def expand() {
+  @Test def expandAll() {
```

Comment:
@systay I'm not sure how to generate a query that uses `expandInto` but we should try and get some more of the operators into the docs. Would you or one of the other cypher crew be able to pair with me for some of Thursday or Friday to have a look at that?